universite-du-quebec-a-montreal.csl

```diff
@@ -147,29 +184,60 @@
                   <text variable="URL" prefix="de "/>
                 </group>
               </if>
-              <else-if type="entry-encyclopedia" match="any">
+              <else-if type="entry-encyclopedia dataset" match="any">
                 <group prefix="Récupéré ">
                   <date form="text" variable="accessed" prefix="le " suffix=" ">
                     <date-part name="day"/>
                     <date-part name="month"/>
                     <date-part name="year"/>
                   </date>
-                  <text variable="URL" prefix="de "/>
+                  <group delimiter=" " prefix="de ">
+                    <text variable="source" suffix=" "/>
+                    <text variable="URL"/>
+                  </group>
                 </group>
               </else-if>
               <else>
                 <choose>
                   <if type="article-journal" match="all">
```

Comment:
why not just continue these with else-if. Also, does the style really need _that_ many different URL forms? Can't we simplify this?

Reply:
Correction made.